web/src/app/alerts/CreateAlertDialog/index.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react'
+import {
+  Dialog,
+  Stepper,
+  Step,
+  StepLabel,
+  DialogContent,
+} from '@material-ui/core'
+import { isWidthDown } from '@material-ui/core/withWidth'
+import DialogNavigation from './DialogNavigation'
+import StepContent from './StepContent'
+import { FormContainer, Form } from '../../forms'
+import DialogTitleWrapper from '../../dialogs/components/DialogTitleWrapper'
+import useWidth from '../../util/useWidth'
+import useCreateAlerts from './useCreateAlerts'
+import { fieldErrors } from '../../util/errutil'
+
+export default props => {
+  const width = useWidth()
+
+  const [activeStep, setActiveStep] = useState(0)
+  const [formFields, setFormFields] = useState({
+    // data for mutation
+    summary: '',
+    details: '',
+    selectedServices: [],
+
+    // form helper
+    searchQuery: '',
+  })
+
+  const [
+    createAlerts,
+    { data: alertsCreated, error: alertsFailed, loading: isCreatingAlerts },
+  ] = useCreateAlerts(formFields, setActiveStep)
+
+  const onStepContentChange = e => {
+    setFormFields(prevState => ({ ...prevState, ...e }))
+  }
+
+  const steps = ['Alert Info', 'Service Selection', 'Review', null]
+
+  const onLastStep = () => activeStep === steps.length - 1
+
+  const onClose = () => {
+    props.handleRequestClose()
+    // NOTE dialog takes time to fade out
+    setTimeout(() => {
+      setActiveStep(0)
+      setFormFields({
+        summary: '',
+        details: '',
+        selectedServices: [],
+        searchQuery: '',
+      })
+    }, 1000)
```

Comment:
Let's find a better way than `setTimeout`. This should only be rendered/exist if it's open so we shouldn't need to reset it on close.

Reply:
We do not do this throughout the app. If a form is closed, on opening a new form, it would be empty